### ISSUE TITLE
CLI enhancement: use folder as project name in package.json instead of "basic"

### ISSUE THREAD

**Issue by fthebaud** [tinyhttp/tinyhttp#263]
When creating a new project using the basic template:
```
> tinyhttp new basic my-app 
Creating a new tinyhttp project from basic template in my-app folder ⚡
Fetching template contents ⌛
Done! You can now launch your project with running these commands:
    
    cd my-app  

    pnpm run start
```

This generates a package.json with the name "basic":
```
  "name": "basic",
```

Wouldn't it be better to use the folder name instead ?
```
  "name": "my-app",
```

**Comment by talentlessguy** [comment]
feel free to PR this enhancement

it's quite easy to do

**Comment by paolotiu** [comment]
Hello! I successfully implemented this in my local fork, but I'm having trouble writing the test for it.

**Comment by talentlessguy** [comment]
closed via #264

**Comment by talentlessguy** [comment]
released in 1.3.3
